test/common.bash

```diff
@@ -31,3 +31,13 @@ create_simple_repo() {
 if [[ -n $COVERAGE ]]; then
     export PATH="$BATS_TEST_DIRNAME/coverage-shim:$PATH"
 fi
+
+if [[ -n "$GIT" ]]; then
+    if [[ -d "$GIT" || ! -x "$GIT" ]]; then
```

Comment:
What's the advantage over simply let the user export PATH that works for any program...?

Reply:
Good point... I wanted to make sure we could allow git binaries with alternate names, but now I think that would be very unlikely, as it is just easier to have it installed in another prefix...

Reply:
Yeah. When I need to use git from its repo I just install it with a  temporary prefix and set the PATH accordingly

Reply:
I'm closing this then. Thanks for the feedback!